src/sardana/macroserver/macros/standard.py

```diff
@@ -676,6 +676,12 @@ def run(self, integ_time):
         for line in table.genOutput():
             self.output(line)
 
+        # Prepare data result to be extracted by getData()
+        self._data = {}
+        self._data['mntGrp'] = self.mnt_grp.name
+        self._data['integ_time'] = integ_time
```

Comment:
I think it is better to use the same type of key naming, either CamelCase or underscore, but always the same. I prefer underscore. So here we could use 'mnt_grp'. What do you think?